### PR TITLE
Added separate utils file for LLM purpose

### PR DESCRIPTION
**Pull request by ark-srivastav**
Relates to #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 1. Check if your `Node.js` version is >= **14**.
 2. Run `npm install` to install the dependencies.
 3. replace your GEMINI api key in `.env.template` and rename the file  to `.env`
-<!-- 3. you need to either be running a local model on `http://localhost:1234` or change the openAI config in `src/utils.js` -->
+<!-- 3. you need to either be running a local model on `http://localhost:1234` or change the openAI config in `src/llm-utils.js` -->
```
Comment (aj47) [inline README.md:14]: please just delete this commented line 14

File: README.md [hunk 2]
```diff
@@ -38,12 +38,12 @@ The main source files are:
 - `src/popup/popup.jsx` The popup window that shows when you press the extension icon
 - `src/background/background.js` Persists and facilitates planning
 - `src/contentScript/contentScript.js` Executes on web pages, executes actions and scrapes elements
-- `src/utils.js` Helper functions
+- `src/llm-utils.js` Helper functions
```
Comment (aj47) [inline README.md:41]: should have a line for `utils.js` and `llm-utils.js` as both files are used now

**Review by aj47 — CHANGES_REQUESTED** [review]
Looks great! Thanks for doing this. I just suggested some small changes

**Review by aj47 — APPROVED** [review]
Thank you!

File: README.md (hunk 1 above)
Comment (ark-srivastav) [reply README.md]: cool, done

File: README.md (hunk 2 above)
Comment (ark-srivastav) [reply README.md]: Yes, good catch, thanks. I've added one more line for LLM utils
``` `src/llm-utils.js` LLM helper functions```

**Pull request merged**